powershell-adapter/Tests/powershellgroup.resource.tests.ps1

```diff
@@ -70,10 +70,11 @@ Describe 'PowerShell adapter resource tests' {
         $r = "{'Name':'TestClassResource1','Prop1':'ValueForProp1'}" | dsc resource set -r 'TestClassResource/TestClassResource' -f -
         $LASTEXITCODE | Should -Be 0
         $res = $r | ConvertFrom-Json
-        $res.afterState.result | Should -Not -BeNull
+        $res.afterState.Prop1 | Should -BeExactly 'ValueForProp1'
+        $res.changedProperties | Should -BeNullOrEmpty
     }
 
-    It 'Export works on PS class-based resource' {
+    It 'Export works on PS class-based resource' -Pending {
```

Comment:
Is this `-Pending` because of the discovery work that's still needed? Can we add a comment / `TODO` here as well and for L.94?

Reply:
Yes, it's pending as there will be a follow-up PR to address discovery for capabilities instead of first one.  I don't think TODO is needed as `Pending` means TODO and I plan to start on it after this is merged.